app/src/main/java/com/testtask/apptesttask/model/interactor/characters/CharactersInteractor.kt

```diff
@@ -2,4 +2,19 @@ package com.testtask.apptesttask.model.interactor.characters
 
 import javax.inject.Inject
 
-class CharactersInteractor @Inject constructor()
\ No newline at end of file
+class CharactersInteractor @Inject constructor(
+    /**TODO characters interactor (Add repository).*/
+) {
+
+    //TODO characters interactor (Add implementation get characters).
+    fun getCharacters() {}
+
+    //TODO characters interactor (Add implementation get favorites characters).
+    fun getFavoritesCharacters() {}
+
+    //TODO characters interactor (Add implementation set favorites characters).
+    fun setFavoritesCharacters() {}
+
+    //TODO characters interactor (Add implementation get favorites characters).
+    fun favorFavoritesCharacters() {}
```

Comment:
What is the difference between `setFavoritesCharacters` and `favorFavoritesCharacters`?